src/Inventory/Asset/MainAsset.php

```diff
@@ -702,6 +702,26 @@ public function rulepassed($items_id, $itemtype, $rules_id, $ports_id = 0)
             $this->setNew();
         }
 
+        //update val according to commonDBTM process (rule)
+        if ($itemtype != NetworkEquipment::getType() && $itemtype != Printer::getType()) {
+            foreach ($this->item->fields as $onadd_key => $onadd_value) {
+                if ($this->isNew() && $default_states_id != '-1' && $onadd_key == 'states_id') {
```

Comment:
```suggestion
                if ($default_states_id != '-1' && $onadd_key == 'states_id') {
```

Both ```add``` and ```update``` must be ignored  ```states_id``` if needed